### ISSUE TITLE
Issue with the child script ROBOT

### ISSUE THREAD

**Issue by romarcg** [peveloper/bachelor-project#4]
Hi,

I have followed the steps until the simulation launching.

However, there is an error when launching the run_simulation script. 

The terrain is loaded and the robot model appears as on it, but there is a yellow error message at the bottom of vrep window with the text:

`
"[string "SCRIPT ROBOT"]:2: in main chunk
`
and when I explored the ROBOT object, in the Scene hierarchy panel, I can see the that the loading of such script looks for an file in a very specific path:

```
local f = loadfile('/Users/stefanopeverelli/Documents/usi/6ths/BachelorProject/scripts/controller/robot.lua')
return f()

```

I tried to look for such call in all the scripts you provided in the scripts folder, but I can't find this call. Where can I change this path?

There is a piece of code in main.lua script:

```
    -- Load the offroad mantra model with its associated script (controller)
    model_path = ABS_PATH .. '../models/offroad.ttm'
    simLoadModel(model_path)
    robot_handle = simGetObjectHandle('ROBOT')

```
is the path is embedded in offroad.ttm (a binary file) ?

Thanks in advance,

**Comment by romarcg** [comment]
I have figured it out. The problem is in the model of the robot `/models/offroad.ttm`. The script defining this model (that links to the controller) is directing to a fixed path `/Users/stefanopeverelli/Documents/usi/6ths/BachelorProject/scripts/controller/robot.lua`.

Vrep can help to change this. The solution is to open the model `offlroad.ttm` and double click on the little note icon besides the root of this model. When the editor opens, we change the path to the corresponding address of our `robot.lua` script. We close the editor.

Then, to be sure that this will be save as a model we open `tools-> object settings` and in the `common` tab we verified that `Object is a model` is checked. 

Finally, we do `File->Save model as` and overwrite (or write to a different) the file `offroad.ttm`.

That fix this issue.
